src/models/sqlOutputContentProvider.ts

```diff
@@ -474,14 +477,18 @@ export class SqlOutputContentProvider {
                     this._queryResultWebviewController
                         .getQueryResultState(uri)
                         .messages.push(message);
-                    this._queryResultWebviewController.getQueryResultState(
-                        uri,
-                    ).tabStates.resultPaneTab = QueryResultPaneTabs.Messages;
-                    this._queryResultWebviewController.state =
-                        this._queryResultWebviewController.getQueryResultState(
-                            uri,
-                        );
-                    vscode.commands.executeCommand("queryResult.focus");
+
+                    // Set state for messages at fixed intervals to avoid spamming the webview
```

Comment:
@aasimkhan30 think this could be solved at a more generic level in the webviews by adding a debounce in the right spot?